data-plane/dispatcher/src/test/java/dev/knative/eventing/kafka/broker/dispatcher/integration/UnorderedConsumerTest.java

```diff
@@ -142,16 +142,12 @@ public void testUnorderedConsumer(final Vertx vertx, final VertxTestContext cont
     final var events = consumerRecords.stream()
       .map(ConsumerRecord::value)
       .toArray(CloudEvent[]::new);
-    final var partitionKeys = Arrays.stream(events)
-      .map(PartitionKey::extract)
-      .collect(Collectors.toList());
```

Comment:
Why this removal?

Reply:
I removed this class `PartitionKey`. Should we put it back?

Reply:
Nono, it's ok that we removed it

Reply:
but you should keep the test as is

Reply:
This change is fine https://github.com/knative-sandbox/eventing-kafka-broker/pull/445/files#diff-80fb1392a05875d0f3d92f4180a8d9392364a1d274dfcf8ecca7b12e41f8fc8fR145 but maybe you need a null check?